lib/cirro_io_v2/resources/gig_ivitation.rb

```diff
@@ -0,0 +1,18 @@
+module CirroIOV2
+  module Resources
+    class GigInvitation < Base
+      def list(user_id: nil, gig_id: nil, before: nil, after: nil, status: nil, limit: 10)
```

Comment:
here the same with `query_params = {}`

If you want to validate you could add a constant listing allowed param keys e.g.:

```
ALLOWED_PARAMS = [:user_id, :gig_id, :limit, :before, :after, :status]

raise ParamNotAllowed if (query_params.keys - ALLOWED_PARAMS).any?
```